gms/feature_service.cc

```diff
@@ -65,20 +65,7 @@ feature_config::feature_config() {
 }
 
 feature_service::feature_service(feature_config cfg) : _config(cfg)
-        , _range_tombstones_feature(*this, features::RANGE_TOMBSTONES)
-        , _large_partitions_feature(*this, features::LARGE_PARTITIONS)
```

Comment:
What keeps sending the feature string?

Reply:
This guy: https://github.com/scylladb/scylla/blob/f3ce5f0cbb140235e6aafa1b140adec92304ceb2/gms/feature_service.cc#L160-L195 . (the same guy responsible of having to copy-paste every new feature once again every time one is added - we should consider applying some kind of a C++ trick to deduplicate this definition)

Reply:
Well, now it's no longer a duplicate since we have some features that are sent and listened to, and some that are only sent.

I suppose we could have a forced_on_feature (with no methods) instead of all the features you eliminated, and then we'll have them in one place. But that's for another time.